templates/es-curator-config.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if .Values.curator.enabled }}
```

Comment:
You could suppress newlines by using {{- <something> -}} tags.

Reply:
Of course, thanks